docs/targets/googlecloudstorage.md

```diff
@@ -0,0 +1,46 @@
+# Event Target for GoogleCloudStorage 
+This event target integrates with GoogleCloudStorage, using received Cloud Event messages to create Blob items.
```

Comment:
Should we link to the Google Cloud Storage webpage? Also it looks like this should be separated not one long word.